core/divelist.c

```diff
@@ -1446,6 +1446,7 @@ void clear_dive_file_data()
 {
 	while (dive_table.nr)
 		delete_single_dive(0);
+	current_dive = NULL;
```

Comment:
the commit message seems wrong, though. Or at least it confuses me.
In general, we expect current_dive to be non-NULL, right?

Reply:
Better?